src/solver/conv_multipass_wino3x3WrW.cpp

```diff
@@ -381,15 +381,33 @@ bool ConvWinograd3x3MultipassWrW<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>
         if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X3{}) ||
            params.kernel_stride_h == 1)
             return false;
-    if(wino_data_tile == 3 && wino_filter_tile == 4)
-        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X4{}))
-            return false;
-    if(wino_data_tile == 3 && wino_filter_tile == 5)
-        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X5{}))
-            return false;
-    if(wino_data_tile == 3 && wino_filter_tile == 6)
-        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X6{}))
-            return false;
+
+    const std::string name = params.GetStream().GetDeviceName();
+    if(params.IsFp16() && StartsWith(name, "gfx908") && params.kernel_stride_h == 1)
```

Comment:
I do not recommend limiting workaround with stride 1 only. @daniellowell what is your opinion?

Reply:
All fp16. I haven’t narrowed it down to stride 1.